src/sinks/aws_kinesis_firehose/integration_tests.rs

```diff
@@ -77,17 +77,18 @@ async fn firehose_put_records() {
         auth: Some(ElasticsearchAuth::Aws(AwsAuthentication::Default {
             load_timeout_secs: Some(5),
         })),
-        endpoint: elasticsearch_address(),
+        endpoint: elasticsearch_address().into(),
         bulk: Some(BulkConfig {
             index: Some(stream.clone()),
             action: None,
         }),
         aws: Some(region),
         ..Default::default()
     };
-    let common = ElasticsearchCommon::parse_config(&config)
+    let common = ElasticsearchCommon::parse_endpoints(&config)
         .await
-        .expect("Config error");
+        .expect("Config error")
+        .remove(0);
```

Comment:
It isn't very clear why the first index needs to be removed. I'm also seeing this same pattern quite a bit in the changes.
Would probably be nice to create a helper function that does this, and is documented.

Reply:
It's also unclear to me why we're doing that everywhere 🤔

Reply:
Before, there was only one common, now there could be many, one for each endpoint. So at tests that use only one endpoint `remove(0)` was added.

I'll add it, we can also assert in there that there is indeed only one endpoint.

Reply:
Ah, gotcha that makes complete sense now 😄